### PR TITLE
styling-error-pages-#47

### LINKED ISSUE #47: Styling error pages

### PR DESCRIPTION
**Pull request by Snieezy**
Closes #47

### CODE REVIEW

File: app/views/errors/validation_error.html.slim [hunk 1]
```diff
@@ -1,3 +1,11 @@
-h1 The change you wanted was rejected.
-h2 Maybe you tried to change something you didn't have access to.
-p If you are the application owner check the logs for more information.
+.flex-container
+  .flex-item
+    .validation_error_logo
+  .flex-item
+    h2 The change you wanted was rejected.
+    hr/
```
Comment (bpietraga) [inline app/views/errors/validation_error.html.slim]: unnecessary `/`

File: app/views/errors/not_found.html.slim [hunk 2]
```diff
@@ -1,3 +1,11 @@
-h1 The page you were looking for doesn't exist.
-p You may have mistyped the address or the page may have moved.
-p If you are the application owner check the logs for more information.
+.grid.mobile-grid-stack
+  .grid-item class="2/5 mobile-1/1 tablet-1/1"
+    .not_found_logo 
+  .grid-item class="3/5 mobile-1/1 tablet-1/1"
+    h2 The page you were looking for doesn't exist.
+    hr/
```
Comment (bpietraga) [inline app/views/errors/not_found.html.slim]: And here `/`

File: app/views/errors/validation_error.html.slim (hunk 1 above)
Comment (Snieezy) [inline app/views/errors/validation_error.html.slim]: This is the old version. I'm currently working on 404, once I will make it working, I will change 422.

**Comment by bpietraga** [comment]
+1

File: app/views/errors/validation_error.html.slim [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
-h2 Maybe you tried to change something you didn't have access to.
-p If you are the application owner check the logs for more information.
+.grid.mobile-grid-stack.grid-middle
+  .grid-item class="2/5 mobile-1/1 tablet-1/1"
+    .my-svg-container
+      object.my-svg.validation_error_logo
+  .grid-item class="3/5 mobile-1/1 tablet-1/1"
+    h2 The change you wanted was rejected.
+    hr
+    p
```
Comment (bpietraga) [inline app/views/errors/validation_error.html.slim]: why not just 'h4` instead od `p`

File: app/views/errors/not_found.html.slim [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
-p You may have mistyped the address or the page may have moved.
-p If you are the application owner check the logs for more information.
+.grid.mobile-grid-stack.grid-middle
+  .grid-item class="2/5 mobile-1/1 tablet-1/1"
+    .my-svg-container
+      object.my-svg.not_found_logo
+  .grid-item class="3/5 mobile-1/1 tablet-1/1"
+    h2 The page you were looking for doesn't exist.
+    hr
+    p
```
Comment (bpietraga) [inline app/views/errors/not_found.html.slim]: why not just `h4` instead of `p`

**Comment by bpietraga** [comment]
+1

**Comment by gotar** [comment]
squash and you can merge

**Pull request merged**